### PR TITLE
Add new endpoint to verify password reset confirm token

### PR DESCRIPTION
**Pull request by mohmyo**
Closes https://github.com/jazzband/dj-rest-auth/issues/65

### CODE REVIEW

**Comment by iMerica** [comment]
On further reflection, I don't think this change belongs in the library. It's definitely useful, but not every thing that is useful belongs in here. 

It's okay for developers to implement their own endpoints for things like this. Especially since its so easy.

**Comment by iMerica** [comment]
Thank you for submitting this PR nonetheless @mohmyo. It's a good reference that shows other devs to do password reset.

**Comment by mohmyo** [comment]
Thank you @iMerica  for your replay, if you change your mind I suggest taking a look for this example with another popular package which implemented this feature.

https://github.com/anexia-it/django-rest-passwordreset#endpoints

I find this has the same point as JWT verify token endpoint.

File: dj_rest_auth/views.py [hunk 1]
```diff
@@ -231,6 +232,26 @@ def post(self, request, *args, **kwargs):
...
+    """
+    serializer_class = PasswordResetConfirmTokenVerifySerializer
+    permission_classes = (AllowAny,)
+
+    def post(self, request, *args, **kwargs):
+        # Create a serializer with request.data
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        serializer.save()
```
Comment (mohmyo) [inline dj_rest_auth/views.py:250]: this line should be removed

**Pull request closed without merging**